tests/integration/test_interaction.py

```diff
@@ -122,7 +122,12 @@ def move_item_to_zone(self, item_value, zone_id, action_key):
     def send_input(self, item_value, value):
         element = self._get_item_by_value(item_value)
         self.wait_until_visible(element)
-        element.find_element_by_class_name('input').send_keys(value)
+        # Since virtual-dom may be updating DOM elements by replacing them completely, the
+        # following method must be used to wait for the input box to appear:
+        textbox_visible_selector = '.numerical-input[style*="display: block"] input'
```

Comment:
@bradenmacdonald Would it be possible to use [`is_displayed()`](http://selenium-python.readthedocs.org/api.html#selenium.webdriver.remote.webelement.WebElement.is_displayed) instead of the style selector?